test/providers.test.js

```diff
@@ -1,6 +1,20 @@
 
 const nearlib = require('../lib/index');
 
+test('json rpc fetch node status', async () => {
+    const config = Object.assign(require('./config')(process.env.NODE_ENV || 'test'));
```

Comment:
looks like there is common setup (`config`, `provider`) that you want to move to `beforeEach`